.github/workflows/ci.yaml

```diff
@@ -54,24 +54,45 @@ jobs:
         environment-file: devtools/conda-envs/test_env.yaml
         python-version: ${{ matrix.python-version }}
         mamba-version: "1.5.10"
-  
-    - name: Install AmberTools
-      if: matrix.ambertools == true
+
+    - name: Set up pixi environment (conda replacement)
+      if: matrix.mamba-executable == 'pixi'
+      uses: prefix-dev/setup-pixi@v0.7.0
+      with:
+        manifest-path: devtools/pixi-envs/pyproject.toml
+        pixi-version: v0.32.1
```

Comment:
I'm curious why you have to use an older version (I see 0.34 as the most recent online https://prefix.dev/channels/conda-forge/packages/pixi)? I'm pretty sure Dependabot will update the version of the action (i.e. `@v0.7.0` above) but not versions in these lines

Reply:
We should be able to use a more recent version, that was just the most recent version when I tested all of it. Would you prefer to go with something like `latest`?

Reply:
Updated both the version of the action `prefix-dev/setup-pixi` as well as the `pixi-version`. Hope you are fine with this 👍